### DISCUSSION TITLE
Ubuntu Variant?

### DISCUSSION THREAD

**Discussion by isaacrlevin** [jlesage/docker-jdownloader-2#220] [General]
I tried to take your base gui image and create an Ubuntu variant of this, so I can install NordVPN for reconnect but ran into al sorts of Docker build issues. Have you tried making an Ubuntu version of this docker image?

**Comment by jlesage** [comment]
You don't need to install VPN software inside the container. Did you try gluetun (https://github.com/qdm12/gluetun) ?

**Reply by isaacrlevin** [reply]
There isn't an easy way for me to trigger a new server with gluten other than restarting the container

**Comment by isaacrlevin** [comment]
If anyone is curious, I decided to roll a "custom solution" to this problem. This is what I have.

- Create a glutun container that the jdownloader container runs through
- Wrote a sh script that creates a file in the `/config` folder when ran
- Set an External Batch Reconnect task in JDownloader that runs that script
 
![image](https://github.com/user-attachments/assets/5fc204e2-807b-4581-b2dd-0921fec7f184)

- Created a bash script that runs on the host to check for that file and if it exists restart the glutun and jdownloader containers using docker compose as well as delete the file
- Have this script run every 5 minutes using cron

Here is a gist of the entire solution
https://gist.github.com/isaacrlevin/b19622d35e8f174dd796247728693ac2
